lib/tokenize.cpp

```diff
@@ -1811,7 +1811,7 @@ namespace {
             return false;
         }
 
-        if (Token::Match(tok1->tokAt(-1), "class|struct|union|enum")) {
+        if (Token::Match(tok1->tokAt(-1), "class|struct|union|enum|namespace")) {
             // fixme
```

Comment:
Just curious: What exactly has to be fixed here?

Reply:
Not sure.  Maybe it was missing namespace?  I think I wrote that a few years ago and I don't remember why.